scripts/seastar-json2code.py

```diff
@@ -52,10 +52,10 @@
 
 spacing = "    "
 def getitem(d, key, name):
-    if key in d:
+    try:
```

Comment:
Maybe
```python
ret = d.get(key)
if ret:
    return ret
else:
    raise Exception(...)
```
?

Reply:
okay. changed as suggested. but i think more idiomatic way is to use exception for error checking.